--- gtk-doc-1.30/gtkdoc/highlight.py.orig	2019-05-08 21:03:58.000000000 +0200
+++ gtk-doc-1.30/gtkdoc/highlight.py	2019-05-15 22:08:04.921285203 +0200
@@ -47,6 +47,7 @@
 
 
 def append_style_defs(css_file_name):
-    os.chmod(css_file_name, stat.S_IWRITE)
+    st = os.stat(css_file_name)
+    os.chmod(css_file_name, st.st_mode | stat.S_IWRITE)
     with open(css_file_name, 'at', newline='\n', encoding='utf-8') as css:
         css.write(HTML_FORMATTER.get_style_defs())
