diff -urN gtk-doc-1.14/configure.ac gtk-doc-1.14.new//configure.ac
--- gtk-doc-1.14/configure.ac	2010-01-03 13:11:00.000000000 +0100
+++ gtk-doc-1.14.new//configure.ac	2010-03-29 16:31:44.160813374 +0200
@@ -27,7 +27,6 @@
 
 AC_ISC_POSIX
 AC_PROG_CC
-AC_PROG_LIBTOOL
 
 dnl Make sure we have pkg-config >= 0.19, so installing in $(datadir) is OK.
 PKG_PROG_PKG_CONFIG([0.19])
@@ -185,23 +184,6 @@
 gtkdoc-common.pl
 help/Makefile
 help/manual/Makefile
-tests/Makefile
-tests/gobject/Makefile
-tests/gobject/src/Makefile
-tests/gobject/docs/Makefile
-tests/gobject/docs-tmpl/Makefile
-tests/bugs/Makefile
-tests/bugs/src/Makefile
-tests/bugs/docs/Makefile
-tests/annotations/Makefile
-tests/annotations/src/Makefile
-tests/annotations/docs/Makefile
-tests/fail/Makefile
-tests/fail/src/Makefile
-tests/fail/docs/Makefile
-tests/empty/Makefile
-tests/empty/src/Makefile
-tests/empty/docs/Makefile
 ])
 
 dnl run chmod on these after parsing them.
diff -urN gtk-doc-1.14/Makefile.am gtk-doc-1.14.new//Makefile.am
--- gtk-doc-1.14/Makefile.am	2009-12-18 09:17:37.000000000 +0100
+++ gtk-doc-1.14.new//Makefile.am	2010-03-29 16:32:00.657469792 +0200
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 ACLOCAL_AMFLAGS=-I m4
 
-SUBDIRS = help tests
+SUBDIRS = help
 
 bin_SCRIPTS = \
 	gtkdoc-check    \
