--- gtk-doc-1.36.0/buildsystems/cmake/meson.build	2026-03-10 11:57:12.000000000 +0100
+++ gtk-doc-1.36.0/buildsystems/cmake/meson.build.orig	2026-03-24 19:44:24.633798087 +0100
@@ -6,7 +6,7 @@ configure_file(
   output: 'GtkDocConfigVersion.cmake',
   configuration: cmake_in,
   install: true,
-  install_dir: get_option('libdir') / 'cmake' / 'GtkDoc',
+  install_dir: get_option('datadir') / 'cmake' / 'GtkDoc',
 )
 
 gtkdoc_cmake_data = [
@@ -16,5 +16,5 @@ gtkdoc_cmake_data = [
 
 install_data(
   gtkdoc_cmake_data,
-  install_dir: get_option('libdir') / 'cmake' / 'GtkDoc',
+  install_dir: get_option('datadir') / 'cmake' / 'GtkDoc',
 )
