--- gtk-doc-1.25/gtkdoc-scangobj.in.orig	2015-06-17 10:23:31.000000000 +0200
+++ gtk-doc-1.25/gtkdoc-scangobj.in	2017-05-25 19:00:21.900251236 +0200
@@ -1269,7 +1269,7 @@
 #endif
 
 #ifdef GTK_IS_WIDGET_CLASS
-#if GTK_CHECK_VERSION(2,1,0)
+#if GTK_CHECK_VERSION(2,1,0) && !GTK_CHECK_VERSION(3,89,2)
     if (!style_prop && GTK_IS_WIDGET_CLASS (class)) {
       properties = gtk_widget_class_list_style_properties (GTK_WIDGET_CLASS (class), &n_properties);
       style_prop = TRUE;
