--- gtk-doc-1.22/configure.ac.noarch	2015-05-08 18:46:08.040460393 +0200
+++ gtk-doc-1.22/configure.ac	2015-05-08 18:46:35.954031830 +0200
@@ -24,10 +24,6 @@ AM_SILENT_RULES([yes])
 # Check for programs
 AC_PROG_CC
 
-# Initialize libtool
-LT_PREREQ([2.2])
-LT_INIT
-
 dnl Make sure we have pkg-config >= 0.19, so installing in $(datadir) is OK.
 PKG_PROG_PKG_CONFIG([0.19])
 
@@ -260,22 +256,6 @@ gtk-doc.cat
 gtkdoc-common.pl
 help/Makefile
 help/manual/Makefile
-tests/Makefile
-tests/gobject/Makefile
-tests/gobject/src/Makefile
-tests/gobject/docs/Makefile
-tests/bugs/Makefile
-tests/bugs/src/Makefile
-tests/bugs/docs/Makefile
-tests/annotations/Makefile
-tests/annotations/src/Makefile
-tests/annotations/docs/Makefile
-tests/fail/Makefile
-tests/fail/src/Makefile
-tests/fail/docs/Makefile
-tests/empty/Makefile
-tests/empty/src/Makefile
-tests/empty/docs/Makefile
 ])
 
 dnl run chmod on these after parsing them.
--- gtk-doc-1.22/Makefile.am.noarch	2015-05-08 18:46:15.247188483 +0200
+++ gtk-doc-1.22/Makefile.am	2015-05-08 18:46:44.464104524 +0200
@@ -1,7 +1,7 @@
 ## Process this file with automake to produce Makefile.in
 ACLOCAL_AMFLAGS=-I m4 ${ACLOCAL_FLAGS}
 
-SUBDIRS = help tests
+SUBDIRS = help
 
 bin_SCRIPTS = \
 	gtkdoc-check    \
