--- gtk-doc-1.28/configure.ac.orig	2018-03-24 16:05:40.000000000 +0100
+++ gtk-doc-1.28/configure.ac	2018-03-24 19:16:15.687516634 +0100
@@ -55,7 +55,7 @@ fi
 
 dnl check for DocBook DTD and stylesheets in the local catalog.
 JH_CHECK_XML_CATALOG([-//OASIS//DTD DocBook XML V4.3//EN], [DocBook XML DTD V4.3])
-JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl], [DocBook XSL Stylesheets])
+JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl-ns/current/html/chunk.xsl], [DocBook XSL Stylesheets])
 
 dnl
 dnl Check for syntax highlighters
--- gtk-doc-1.28/gtk-doc.xsl.orig	2018-03-24 19:16:15.684183303 +0100
+++ gtk-doc-1.28/gtk-doc.xsl	2018-03-24 19:17:02.400818509 +0100
@@ -8,7 +8,7 @@
   <!-- http://www.sagehill.net/docbookxsl/Chunking.html#FastChunking says we should use
        "chunkfast.xsl", but I can't see a difference. Also it is not clear wheter
        one should still set "chunk.fast". Again, no meassuable difference. -->
-  <xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl"/>
+  <xsl:import href="http://docbook.sourceforge.net/release/xsl-ns/current/html/chunk.xsl"/>
   <xsl:include href="devhelp2.xsl"/>
   <xsl:include href="version-greater-or-equal.xsl"/>
 
