--- gtk-doc-1.26/gtk-doc.xsl~	2015-06-26 08:05:50.000000000 +0200
+++ gtk-doc-1.26/gtk-doc.xsl	2017-11-26 11:32:55.545442969 +0100
@@ -7,7 +7,7 @@
   <!-- import the chunked XSL stylesheet -->
   <!-- http://www.sagehill.net/docbookxsl/Chunking.html says we should use
        "chunkfast.xsl", but I can see a difference -->
-  <xsl:import href="http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl"/>
+  <xsl:import href="http://docbook.sourceforge.net/release/xsl-ns/current/html/chunk.xsl"/>
   <xsl:include href="devhelp2.xsl"/>
   <xsl:include href="version-greater-or-equal.xsl"/>
 
--- gtk-doc-1.26/configure.ac~	2017-11-26 11:56:42.000000000 +0100
+++ gtk-doc-1.26/configure.ac	2017-11-26 11:57:11.985289031 +0100
@@ -53,7 +53,7 @@
 
 dnl check for DocBook DTD and stylesheets in the local catalog.
 JH_CHECK_XML_CATALOG([-//OASIS//DTD DocBook XML V4.3//EN], [DocBook XML DTD V4.3])
-JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl/current/html/chunk.xsl], [DocBook XSL Stylesheets])
+JH_CHECK_XML_CATALOG([http://docbook.sourceforge.net/release/xsl-ns/current/html/chunk.xsl], [DocBook XSL Stylesheets])
 
 dnl
 dnl Check for syntax highlighters
